jekyll/_cci2/language-python.md

```diff
@@ -2,104 +2,266 @@
 layout: classic-docs
 title: "Language Guide: Python"
 short-title: "Python"
-description: "Building and Testing with Python on CircleCI 2.0"
+description: "Continuous integration and deployment with Python on CircleCI"
 categories: [language-guides]
 order: 7
 ---
 
-This document describes the [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file for a sample application written in Python. 
+This document describes
+how to configure CircleCI
+using a sample application written in Python.
 
 * TOC
 {:toc}
 
-## Demo Python Django Reference Project
+## Overview
 
-We maintain a reference Python Django project to show how to build Django on CircleCI 2.0:
+This guide uses a [sample Django application](https://github.com/CircleCI-Public/circleci-demo-python-django)
+to describe configuration best practices
+for Python applications building on CircleCI.
+Consider [forking the repository](https://help.github.com/articles/fork-a-repo/)
+and rewriting the configuration file
+as you follow this guide.
 
-- <a href="https://github.com/CircleCI-Public/circleci-demo-python-django" target="_blank">Demo Python Django Project on GitHub</a>
-[Demo Python Django Project building on CircleCI](https://circleci.com/gh/CircleCI-Public/circleci-demo-python-django){:rel="nofollow"}
+## Configuration Walkthrough
 
-In the project you will find a commented CircleCI configuration file <a href="https://github.com/CircleCI-Public/circleci-demo-python-django/blob/master/.circleci/config.yml" target="_blank">`.circleci/config.yml`</a>. This file shows best practice for using CircleCI 2.0 with Python projects.
```

Comment:
Add back this link as well

Reply:
Since there is a copy of the full configuration file in this guide, how does adding a link to the same configuration help the reader?

Reply:
Both are helpful, they serve different readers, a full config file doesn't replace the value of a link to the actual config file. Some readers will be helped by one, others helped by the other, some will need both to put it all together.

Reply:
This doesn't answer my question. The full config file is a copy of the configuration file in the project. There is no "one" or the "other" — it is literally the same YAML.

I am already linking to the project on GitHub. This is the most powerful link in the guide; developers understand how to navigate file directories to read source code. The purpose of the guide is to connect concepts in the configuration to other documentation.

Reply:
Not all of our readers are developers who understand where a . directory is, for sure. You are not aware of the background of all of our developers and neither am I, but this thread has taken much longer than adding back a single link to the config file, lol!